todo_list_server/with_orm/main.py

```diff
@@ -0,0 +1,58 @@
+from typing import List
+
+from fastapi import FastAPI, Response, Request, HTTPException
+from fastapi.exceptions import RequestValidationError
+from fastapi.responses import JSONResponse
+from sqlmodel import Session
+
+from database import create_db_and_tables, engine
+
+import crud
+import models
+
+app = FastAPI()
+
+
+@app.on_event("startup")
+def on_startup():
+    create_db_and_tables()
+
+
+@app.exception_handler(RequestValidationError)
+async def validation_exception_handler(_: Request, exc: RequestValidationError):
+    message = exc.errors()[0]["msg"]
+    return JSONResponse(content={"detail": message}, status_code=400)
+
+
+@app.get(path='/todos', response_model=models.SuccessResponse[List[models.TodoRead]])
+async def read_todos():
```

Comment:
```suggestion
async def read_todos() -> models.SuccessResponse[List[models.TodoRead]]:
```

`response_model`에 명시한 모델을 함수의 리턴 타입으로 정의해두는 것이 좋습니다

Reply:
https://github.com/eunnbi/zero-to-hero/pull/3/commits/3f1c3e14ad146f8aa82bfe08ba6b2ccb7ccaad6e